bin/dktl.sh

```diff
@@ -76,7 +76,14 @@ else
     if [ -z "$VENDOR" ]; then
```

Comment:
Check this logic, seems to always run composer install

Reply:
I saw this when we were working together, but I was not able to replicate. When the vendor directory is present, composer install does not run.